tests/e2e/convolution/CMakeLists.txt

```diff
@@ -0,0 +1,169 @@
+################################################################################
+# Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py from           #
+# tests/e2e/convolution/BUILD.bazel                                            #
+#                                                                              #
+# Use iree_cmake_extra_content from iree/build_defs.oss.bzl to add arbitrary   #
+# CMake-only content.                                                          #
+#                                                                              #
+# To disable autogeneration for this file entirely, delete this header.        #
+################################################################################
+
+iree_add_all_subdirs()
+
+iree_generated_e2e_runner_test(
+  NAME
+    e2e_conv2d_cpu_f32_f32_f32_small
+  TEST_TYPE
+    conv2ds
+  GENERATOR
+    "generate_e2e_conv2d_tests.py"
+  GENERATOR_ARGS
+    "--input_type=f32"
+    "--kernel_type=f32"
+    "--acc_type=f32"
+    "--shapes=small"
+  TEST_RUNNER
+    iree_tools_testing_e2e_iree-e2e-conv2d-test
+  TARGET_BACKENDS
+    "llvm-cpu"
+  DRIVERS
+    "local-task"
+  COMPILER_FLAGS
+
+  LABELS
+
+  TARGET_CPU_FEATURES_VARIANTS
+    "default"
```

Comment:
Something here in the generator could be changed. Empty argument lists can be omitted.

Reply:
done